lightning/src/ln/channel.rs

```diff
@@ -3056,10 +3085,10 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 			}
 		}
 
-		let proposed_sat_per_kw = msg.fee_satoshis  * 1000 / closing_tx.get_weight() as u64;
+		let mut min_feerate_per_kw = 253; // Min relay fee of at least 1 sat/vbyte (rounding up to ensure its never below 1 when divided by 4 to convert weight->vbyte).
```

Comment:
See IRC, I can't come up with a rational for 253. I think we should constify this value and write down a Big Comment around to explain how we come up with this magic value. Mostly in case  of someone changing something in Core to let us remember what's going on.

Reply:
After IRC discussions, there is a consensus that 253 is enough, but we maybe good with a bit under. To be tested. In the meanwhile marking this as resolved.